tartiflette/schema/schema.py

```diff
@@ -105,6 +109,15 @@ def subscription_type(self) -> Optional[str]:
     def subscription_type(self, value: str) -> None:
         self._subscription_type = value
 
+    def get_operation_type(self, operation_name):
```

Comment:
We could one line this method with something like:
```suggestion
    def get_operation_type(self, operation_name):
        return getattr(self, operation_name + "_type", None)
```
I find it less readable but we would be able to reduce code by removing constants `_DEFAULT_***_TYPE` and remove `capitalize` the operation name.

Reply:
At this state of the query execution, we need to go fast. I think getattr will be slower than returning constants.
Also remove the ifs in get_operation_type and use a dict with the constant as values.

Reply:
The goal of `get_operation_type` is to return the root node name of an operation (`query`, `mutation`, `subscription`) which is respectively stored in `self.query_type`, `self. mutation_type ` & `self. subscription_type `.

By default those name are (`Query`, `Mutation`, `Subscription`) but depending on the schema we can have custom names:
```
schema {
    query: CustomRootQuery
}

type CustomRootQuery {
    test: String
}
```
Here we'll have `CustomRootQuery` instead of `Query` (meaning that `self.query_type` will be `CustomRootQuery`).

The fact is that `operation_name` paremeter will be either `Query`, `Mutation` or `Subscription`. That's why I need those `if`s to determine which root node name I have to return.

I don't see how I could have a dict mapping that returns one of those three attributes. At best it could return the attribute name but I'll have to do a `getattr` then.

Reply:
Hum maybe, i'll have to look at it more in depth next, cause we are doing a lot of self.blabla(self.param) wich is stupid, maybe a refac of this whole think will happens.

If #48 is done with this PR. I'd say that it's perfect. Add some functionnal tests (like do a mutation) to validate everythigs and i'll approve the shit out of it !

Reply:
Function test added, sir.